test/Interop/Cxx/members/emit-function-from-static-var-init-irgen.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %target-swift-emit-ir %s -I %S/Inputs -enable-cxx-interop -validate-tbd-against-ir=none | %FileCheck %s
+
+// TODO: See why -validate-tbd-against-ir=none is needed here
```

Comment:
Can you file an SR for this and reference it here?

Reply:
Oops, I see I already made this comment. Sorry. Anyway, can you please update the comments to include the created SR? Thanks!

Reply:
I did that in https://github.com/apple/swift/pull/35056/commits/8bcef1b0f4f96491010ece9c38c1f25556021808 :)

Reply:
Thanks.